ietf/home/templates/home/iab_home_page.html

```diff
@@ -1,58 +1,77 @@
 
 {% block main_content %}
+
 <main id="content">
-    <h1 class="visually-hidden">Welcome to the <abbr title="Internet Engineering Task Force">IETF</abbr></h1>
+    <h1 class="visually-hidden">
+        Welcome to the <abbr title="Internet Engineering Task Force">IETF</abbr>
+    </h1>
     <!--  Jumbotron -->
     {% image self.main_image max-4000x4000 as bg_image %}
-    <section class="jumbotron text-white iab-home_page__jumbotron"
-        style="background-image: url({% if bg_image.url %}{{ bg_image.url }}{% else %}{% static 'img/group-01.jpg' %}{% endif %});">
-        <div class="container container--iab position-relative">
-            <h1 class="iab-h1" >{{ self.heading }}</h1>
+    <section
+        class="jumbotron text-white iab-home_page__jumbotron"
+        style="background-image: url({% if bg_image.url %}{{ bg_image.url }}{% else %}{% static 'img/group-01.jpg' %}{% endif %});"
+    >
+        <div class="container position-relative">
+            <h1 class="iab-h1">{{ self.heading }}</h1>
             {% if self.introduction %}
-                <p class="lead">{{ self.introduction }}</p>
-            {% endif %}
-            {% if self.button_link %}
-                <p>
-                    <a href="{{ self.button_link.url }}"
-                        class="btn iab-btn-secondary text-white mt-4">{{ self.button_text }}</a>
-                </p>
+            <p class="lead">{{ self.introduction }}</p>
+            {% endif %} {% if self.button_link %}
+            <p>
+                <a
+                    href="{{ self.button_link.url }}"
+                    class="btn iab-btn-secondary text-white mt-4"
+                    >{{ self.button_text }}</a
+                >
+            </p>
             {% endif %}
         </div>
     </section>
 
     <!-- Blog Posts -->
-    <div class="container container--iab my-400pc" >
+    <div class="container my-400pc">
         <div class="row">
             <section class="col-md-6 iab-left-column">
                 <h2 class="mb-4">Blog posts</h2>
                 <ul class="list-unstyled mb-200pc">
-                    {% with self.blogs.0 as post %}
-                        {% with True as show_image %}
-                            {% include "includes/post_iab.html" %}
-                        {% endwith %}
+                    <!-- prettier-ignore -->
+                    {% with self.blogs.0 as post %} 
+                        {% with True as show_image %} 
+                            {% include "includes/post_iab.html" %} 
+                        {% endwith %} 
                     {% endwith %}
                 </ul>
-                <a class="btn iab-btn-secondary text-white" href="{{ self.event_index.url }}">All IAB blog posts on IETF website</a>
+                <a
+                    class="btn iab-btn-secondary text-white"
+                    href="{{ self.blog_index.url }}"
+                    >All IAB blog posts on IETF website</a
+                >
```

Comment:
Breaking `</a>` onto two lines seems kinda weird

Reply:
oh i guess it's for symmetry with the broken-into-lines opening tag? still feels weird to me but no biggie 🤷